### PR TITLE
[Docs] update Japanese translation of feature_auto_shift.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/feature_auto_shift.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_auto_shift.md [hunk 1]
```diff
@@ -79,6 +82,41 @@
...
+        case KC_NONUS_BSLASH:
+#    endif
+            return true;
+    }
+    return false;
+}
+```
+この機能はデフォルトで有効になっており、定義する必要はありません。
+
+### AUTO_SHIFT_REPEAT (simple define)
```
Comment (s-show) [inline docs/ja/feature_auto_shift.md:112]: 翻訳漏れだと思います。
```suggestion
### AUTO_SHIFT_REPEAT (簡単な定義)
```

File: docs/ja/feature_auto_shift.md [hunk 2]
```diff
@@ -15,12 +15,15 @@
 
 キーをタップする時に、キーを放す前にほんの短い間押したままにします。この押したままにする時間は全ての人にとって異なる長さです。自動シフトは、定数 `AUTO_SHIFT_TIMEOUT` を定義し、これは普段の押された状態の時間の2倍に通常は設定されます。タイマーは、キーを押す時に開始され、キーを放す時に止まります。押された時間が `AUTO_SHIFT_TIMEOUT` 以上の場合に、キーのシフトバージョンが発行されます。時間が `AUTO_SHIFT_TIMEOUT` 時間よりも短い場合は、通常の状態が発行されます。
```
Comment (s-show) [inline docs/ja/feature_auto_shift.md:16]: 原文が変更されて、訳文の「キーを放す時に止まります」に当たる「then stops when you release the key」が無くなっています。なので、原文の他の変更点と合わせて次のように修正するのはどうでしょうか。
```suggestion
キーをタップする時に、キーを放す前にほんの短い間押したままにします。この押したままにする時間は全ての人にとって異なる長さです。自動シフトは、定数 `AUTO_SHIFT_TIMEOUT` を定義し、これは普段の押された状態の時間の2倍に通常は設定されます。タイマーは、キーを押す時に開始され、`AUTO_SHIFT_TIMEOUT` の時間が過ぎてもキーを放さなかった場合、キーのシフトバージョンが発行されます。時間が `AUTO_SHIFT_TIMEOUT` 時間よりも短い場合は、通常の状態が発行されます。
```

**Review by s-show — APPROVED** [review]

**Review by peepeetee — APPROVED** [review]

**Comment by zvecr** [comment]
Closing as this would have gone stale multiple times by now.

**Pull request closed without merging**